<?xml version="1.0"?>
<launch>
  <arg name="guidance" default="true" />

  <node pkg="jsk_uav_forest_perception" type="mono_convert_16UC1_node" name="mono_convert" output="screen" >
    <param name="mono_topic_name" value="guidance/depth_image"/>
    <param name="cvt_topic_name" value="guidance/depth/image_raw"/>
  </node>


  <!-- Nodelet manager for this pipeline -->
  <node pkg="nodelet" type="nodelet" args="manager"
        name="record_player_manager" output="screen"/>

  <!-- format change -->

</launch>
